install/collect_mac_addresses_for_ncns.md

```diff
@@ -3,24 +3,24 @@
 ## Topics
 
 1. [Set up networking](#1-set-up-networking)
-2. [Collect MAC addresses](#2-collect-mac-addresses)
+1. [Collect MAC addresses](#2-collect-mac-addresses)
```

Comment:
If we include the number in the actual header, I think it makes sense to use the actual number here, as a reminder that the numbering isn't entirely fluid.

```suggestion
2. [Collect MAC addresses](#2-collect-mac-addresses)
```